f600233d1b154d507b782f84604bcb12/update/sys_script_include_97c87b02974611106fa8b4b3f153afef.xml

```diff
@@ -65,6 +66,9 @@ ErrorHandler.prototype = {
 			case this.COST_RESPONSE_ERROR:
 				gs.error(error.message);
 				break;
+			case this.METHOD_ERROR:
```

Comment:
@juliusfitzhugh-ccpo Can you document for posterity the intended use of this new error type?

Reply:
Added a comment above `this.METHOD_ERROR`. See 842488d532983e678481c2eb5b32e9576e82ef6a.